Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/SceneLoaderWrapper.cs

```diff
@@ -181,7 +180,7 @@ void OnSceneEvent(SceneEvent sceneEvent)
                     if (NetworkManager.IsServer)
                     {
                         // Send client RPC to make sure the client stops the loading screen after the server handles what it needs to after the client finished synchronizing, for example character spawning done server side should still be hidden by loading screen.
-                        StopLoadingScreenClientRpc(new ClientRpcParams { Send = new ClientRpcSendParams { TargetClientIds = new[] { sceneEvent.ClientId } } });
+                        ClientStopLoadingScreenRpc(RpcTarget.Group(new[] { sceneEvent.ClientId }, RpcTargetUse.Persistent));
```

Comment:
I would use RpcTargetUse.Temp here.
Persistent would be something you would use if you knew it wouldn't be changing frequently and you do need to dispose.

Temp has a minor alloc hit, but for this context it is perfectly fine.

Reply:
Great, thanks for clarifying.
Done!

Reply:
Oh and thanks for spotting this originally @RikuTheFuffs! 🎉